pkg/quota/set_diskquota.go

```diff
@@ -37,13 +38,15 @@ func processSetQuotaReexec() {
 		}
 	}()
 
-	if len(os.Args) != 3 {
-		err = fmt.Errorf("invalid arguments: %v, it should be: %s: <path> <size>", os.Args, os.Args[0])
+	if len(os.Args) != 4 {
+		err = fmt.Errorf("invalid arguments: %v, it should be: %s: <path> <size> <quota id>", os.Args, os.Args[0])
```

Comment:
This err assignment is useless. I think we need to add a log to record this.

```
logrus.Errorf("invalid arguments: %v, it should be: %s: <path> <size> <quota id>", os.Args, os.Args[0])
```

Reply:
It will return the error to runc and then print it. All the hooks do it like this.